tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1623,6 +1623,278 @@ func(
 
 `;
 
+exports[`mixed-38.php - php-verify: mixed-38.php 1`] = `
+<?php
+$var = 1;
+$var = 2;
+?>
+<div></div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+$var = 1;
+$var = 2;
+?>
+<div></div>
+
+`;
+
+exports[`mixed-39.php - php-verify: mixed-39.php 1`] = `
+<?php
+
+$var = 1;
+$var = 2;
+?>
+<div></div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+
+$var = 1;
+$var = 2;
+?>
+<div></div>
+
+`;
+
+exports[`mixed-40.php - php-verify: mixed-40.php 1`] = `
+<?php
+
+
+$var = 1;
+$var = 2;
+?>
+<div></div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+
+$var = 1;
+$var = 2;
+?>
+<div></div>
+
+`;
+
+exports[`mixed-41.php - php-verify: mixed-41.php 1`] = `
+<?php
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+
+`;
+
+exports[`mixed-42.php - php-verify: mixed-42.php 1`] = `
+<?php
+?>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+?>
+
+`;
+
+exports[`mixed-43.php - php-verify: mixed-43.php 1`] = `
+<?php
+
+?>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+?>
+
+`;
+
+exports[`mixed-44.php - php-verify: mixed-44.php 1`] = `
+<?php
+declare(strict_types=1);
+$var = 1;
+$var = 2;
+?>
+<div></div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+declare(strict_types=1);
+$var = 1;
+$var = 2;
+?>
+<div></div>
+
+`;
+
+exports[`mixed-45.php - php-verify: mixed-45.php 1`] = `
+<?php
+
+declare(strict_types=1);
+
+$var = 1;
+$var = 2;
+?>
+<div></div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+
+declare(strict_types=1);
+
+$var = 1;
+$var = 2;
+?>
+<div></div>
+
+`;
+
+exports[`mixed-46.php - php-verify: mixed-46.php 1`] = `
+<?php
+
+
+declare(strict_types=1);
+
+
+$var = 1;
+$var = 2;
+?>
+<div></div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+
+declare(strict_types=1);
+
+$var = 1;
+$var = 2;
+?>
+<div></div>
+
+`;
+
+exports[`mixed-47.php - php-verify: mixed-47.php 1`] = `
+<?php
+namespace Foo;
+$var = 1;
+$var = 2;
+?>
+<div></div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+namespace Foo;
```

Comment:
Need also fix, we should respect user input. If `newline` exists print this, if not don't add, i will fix it in next PR